### PR TITLE
Add show:storage option to storm runtime

### PR DESCRIPTION
**Pull request by invisig0th**

### CODE REVIEW

File: synapse/lib/snap.py [hunk 1]
```diff
@@ -684,12 +686,16 @@ async def iterStormPodes(self, text, opts, user=None):
...
             dorepr = opts.get('repr', False)
             dopath = opts.get('path', False)
+            show_storage = opts.get('show:storage', False)
 
         async for node, path in self.storm(text, opts=opts, user=user):
 
             pode = node.pack(dorepr=dorepr)
             pode[1]['path'] = await path.pack(path=dopath)
 
+            if show_storage:
```
Comment (Cisphyx) [inline synapse/lib/snap.py:696]: Should we make this an argument on `node.pack` since the node will already have all the sodes in 3.0.0?

Comment (invisig0th) [reply synapse/lib/snap.py:696]: er... `node.getStorNodes()` :)

Comment (Cisphyx) [reply synapse/lib/snap.py:696]: gotcha, yeah that works too

**Review by Cisphyx — APPROVED** [review]

File: docs/synapse/devguides/storm_api.rst [hunk 2]
```diff
@@ -625,6 +625,19 @@ Example:
         # Only include required messages.
         opts = {'show': []}
 
+show:storage
+------------
+
+A boolean option which, when set to ``true``, instructs the Storm runtime to add a ``storage`` key to each yielded node
+which contains a raw breakdown of storage nodes which can be used to determine which parts of the node are stored in
+which layer within the view.
```
Comment (vEpiphyte) [inline docs/synapse/devguides/storm_api.rst:633]: Also need to update the `node` message type.

**Pull request merged**